### ISSUE TITLE
Chocolatey packages should be test in windows container

### ISSUE THREAD

**Issue by efaruk** [chocolatey/chocolatey.org#472]
Hi everyone some of packages not working in windows container. So testing chocolatey packages also in windows container (core, nano) could be a great idea?

**Comment by gep13** [comment]
@efaruk can you provide an example of a package that isn't working?  Is the underlying application "meant" to run on nano/core?

**Comment by efaruk** [comment]
Of course; nano text editor and midnight commander are not working in windows core container...

**Comment by gep13** [comment]
Just to confirm, you are referring to:

https://chocolatey.org/packages/nano

and

https://chocolatey.org/packages/mc

Can you provide a gist of the installation log, and some further information about what exactly `not working` is actually referring to?

**Comment by efaruk** [comment]
Yes i'm referring those.
Right now, i can give quick info: I can tell, installation was successful but nano is just quitting without giving any tip about what just happened and mc is the same. I you need an example of working one: far manager is working quite good so far....

**Comment by gep13** [comment]
Are you saying that if you install those applications directly, I.e. Not using Chocolatey, that they work correctly?

**Comment by efaruk** [comment]
No i never say that, i'm installing those apps only using chocolatey. This is not a chocolatey issue of course. I m just saying if those apps was tested in windows container too it'll be great...
